### PR TITLE
Disable headless mode, fix tests broken on Chrome 71

### LINKED ISSUE #15728: Avoid document-register-element changes in Chrome 67+
Latest version of document-register-element v1.8 features detect Chrome 67 in a different way and it leaves it untouched (100% native Custom Elements V0 + V1).

I see greenkeeper has not been triggering with latest changes but I strongly suggest you to try latest version and use it so that whenever native V1 with custom elements builtins ships on mobile too it won't affect the browser at all.

### LINKED ISSUE #19647: Fix tests failure with Chrome 71

### PR DESCRIPTION
**Pull request by choumx**
Fixes #19647. Fixes and unskips amp-bind tests failing on Chrome 71.

### CODE REVIEW

**Comment by choumx** [comment]
We should run tests before/after on IE and Edge to test:
- https://caniuse.com/#feat=custom-elementsv0
- https://caniuse.com/#feat=custom-elementsv1

**Comment by choumx** [comment]
Blob URLs broken on headless Chrome 71: https://github.com/GoogleChrome/puppeteer/issues/3463

**Comment by choumx** [comment]
Omg, tests passed. @estherkim quick, before my luck runs out.

File: extensions/amp-list/0.1/test/integration/test-amp-list.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ describe('amp-list', function() {
       '</template>' +
     '</amp-list>';
 
-  // TODO(#19647): Unskip tests
+  // TODO(cathyxz, #19647): Fix test on Chrome 71.
```
Comment (choumx) [inline extensions/amp-list/0.1/test/integration/test-amp-list.js]: @cathyxz FYI.

**Comment by rsimha** [comment]
Delayed LGTM. Thanks for investigating this!

**Pull request merged**